docs/remote/devcontainer-cli.md

```diff
@@ -126,7 +138,77 @@ For example, you may want to pre-build several images that you then reuse across
 
 That's it!
 
-### Adding automation
+## Building Multi-Architecture builds
+
+#### Pre-requisites
+1. [Node.js 14+](https://nodejs.org).
+2. [The `docker` CLI](/docs/remote/containers#installation).
+3. [Docker `Buildx`](https://docs.docker.com/buildx/working-with-buildx/)
+4. Docker logged in to a Dockerhub registry.
+
+Ensure that the command `docker buildx -h` returns a valid help message.
+
+Docker needs to be logged in to a container registry because the multi-architecture build defaults to pushing the multi-arch image manifesto to a container registry. If not, the `--no-push` option needs to be used.
+
+#### Process Example
+
+1. Create a basic Dockerfile as below:
+
+    ```Dockerfile
+    #FROM alpine:latest
+    ARG VARIANT=3.13
+    FROM mcr.microsoft.com/vscode/devcontainers/base:0-alpine-${VARIANT}
+    #FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-12
+    CMD echo "Running on $(uname -m)"
+    ```
+
+2. Create a basic `devcontainer.json` as below:
+
+    ```json
+    {
+    "name": "Alpine",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "args": { "VARIANT": "3.15" },
+        "settings": {},
+        "extensions": [],
+        "remoteUser": "vscode"
+      }
+    }
+    ```
+
+3. Place both files in a `.devcontainer` directory.
+
+4. Run the below command:
+
+    ```bash
+    devcontainer build \
+        --platform linux/amd64,linux/arm64 \
+        --image-name hubusername/multiarch-test:v1 \
+        path-to-devcontainer-json
+    ```
+
+#### Breaking down the command
+
+1. `devcontainer build`: As before, this invokes the devcontainer CLI build command.
+2. `--platform linux/amd64,linux/arm64`: This creates Docker images for the `linux/amd64` and `linux/arm64` architectures.
+3. `--image-name hubusername/multiarch-test:v1`: **This is mandatory when using `--platform` and optional otherwise.** This will tag the image and needs to follow container registry naming conventions to upload.
+
+#### No Push
+
+By default, the Devcontainer CLI **will push** to the container registry. To avoid doing this for local development purposes, append `--no-push` to the command as shown below:
```

Comment:
Is this true broadly or only in the multi-platform case?  If it's multi-platform only can we specify that?